src/main/java/uk/gov/companieshouse/reconciliation/config/EmailLinkModel.java

```diff
@@ -1,7 +1,12 @@
 package uk.gov.companieshouse.reconciliation.config;
 
+import uk.gov.companieshouse.reconciliation.model.ResourceLink;
+
 import javax.validation.constraints.NotNull;
 
+/**
+ * Email link model that defines a rank that can be used to order {@link ResourceLink}'s.
```

Comment:
Please remove the apostrophe from ResourceLink's.

Reply:
Resolved in latest commit.